src/syscheckd/src/whodata/win_whodata.c

```diff
@@ -555,9 +555,18 @@ int restore_audit_policies() {
         merror(FIM_ERROR_WHODATA_RESTORE_POLICIES);
         return 1;
     }
+
     // Get the current policies
     char *cmd_output = NULL;
-    const int wm_exec_ret_code = wm_exec(command, &cmd_output, &result_code, 5, NULL);
```

Comment:
Since you are using a variable to avoid a magic number with the retries, at least I would do the same with the timeout value